src/main/java/org/jcodec/api/transcode/Transcoder.java

```diff
@@ -201,6 +199,14 @@ public void finalFlushQueues() throws IOException {
                         }
                     }
                 }
+            } else {
+                for (AudioFrameWithPacket audioFrame : audioQueue) {
```

Comment:
`AudioFrameWithPacket` - maybe lets call it `AudioSampleBuffer` like Apple Core Media does?
https://developer.apple.com/documentation/coremedia/cmsamplebuffer-u71
Basic idea is:
SampleBuffer == data + timing + extra metadata

Reply:
there is a class org.jcodec.common.model.AudioBuffer that contains data + associated audio format information. This Class is the internal of the transcoder to carry 2 pieces, i.e. the decoded AudioBuffer + the packet. I would prefer to leave the name as is, or replace it with Pair (_2) AudioBuffer, Packet. Let me know.